src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
 
     export function requestAnimationFramePolyfill(fn: () => void) {
```

Comment:
`POLYFILL_TIMEOUT_MILLISECONDS` can be read as the "timeout for a polyfill in milliseconds".  However, this isn't the timeout for just a polyfill, it is the timeout for the requestAnimationFrame polyfill.